ddl/db_test.go

```diff
@@ -1760,7 +1760,7 @@ func (s *testDBSuite) TestTableDDLWithTimeType(c *C) {
 	s.testErrorCode(c, "alter table t change column a aa time(7)", tmysql.ErrTooBigPrecision)
 	s.testErrorCode(c, "alter table t change column a aa datetime(7)", tmysql.ErrTooBigPrecision)
 	s.testErrorCode(c, "alter table t change column a aa timestamp(7)", tmysql.ErrTooBigPrecision)
-	s.mustExec(c, "alter table t change column a aa timestamp(0)")
+	s.mustExec(c, "alter table t change column a aa datetime(0)")
```

Comment:
Why this is changed to `datetime` ?

Reply:
because previous test will be failure with 
```
ERROR 1105 (HY000): unsupported modify column type 7 not match origin 12
```
`- -` and it's strange mustExec in ddl_test seems ignore the err and doesn't fail the case.

Reply:
The test doesn't init the testKit, but reuse the one in the suite, so the test case never fail.